e2e/types/types.go

```diff
@@ -95,7 +95,7 @@ type Workload interface {
 	GetPath() string
 	GetBranch() string
 
-	// SupportsDeployer returns tue if this workload is compatible with deployer.
+	// SupportsDeployer returns true if this workload is compatible with deployer.
```

Comment:
This should not be part of this commit. You can add it in a separate commit or send another PR with the fix.

If we revert the tags changes, we would lose the typo fix. This makes it harder to maintain.

Reply:
Sure, will put it in another PR.